### PR TITLE
chore(test data): Update seed script to make lists public

### PR DESCRIPTION
**Pull request by nina-py**
## Goal

Improve ease of Admin Tools development for Shareable Lists.

- Update seed script to make lists public

- And have slugs generated for them so that they can be used in Dev Curation Admin Tools for testing and development

## Tickets

 - not ticketed, but related to OSL - 167

### CODE REVIEW

File: prisma/seed.ts [hunk 1]
```diff
@@ -38,6 +39,14 @@ async function main() {
       userId: randomUser.userId,
     });
 
+    // turn it public so that it obtains a slug - this seed data
+    // is created for the benefit of testing the admin tools after all
+    await updateShareableList(
+      prisma,
+      { externalId: list.externalId, status: ListStatus.PUBLIC },
```
Comment (jpetto) [inline prisma/seed.ts:46]: is there value is keeping some lists private? or is this seed script only in service of the admin tool?

**Review by jpetto — APPROVED** [review]
just one non-blocking question 😄

**Review by katerinachinnappan — APPROVED** [review]

File: prisma/seed.ts (hunk 1 above)
Comment (nina-py) [reply prisma/seed.ts:46]: I think there is, let me try and make only some lists public as I need to see how the component looks in both cases in the app itself, and not just in test coverage.

File: prisma/seed.ts [hunk 2]
```diff
@@ -38,6 +39,16 @@ async function main() {
       userId: randomUser.userId,
     });
 
+    // Turn some lists public so that the status changes and a slug is generated.
+    // This seed data is created for the benefit of testing the admin tools after all.
+    if (Math.random() > 0.5) {
```
Comment (jpetto) [inline prisma/seed.ts:44]: as long as you have faith in `Math.random()` being greater than `0.5` at least once in 50 chances 😅

**Review by jpetto — APPROVED** [review]

File: prisma/seed.ts (hunk 2 above)
Comment (nina-py) [reply prisma/seed.ts:44]: Well, strictly speaking, we're all right even if it's not an even split between public and private lists in seed data, we just need a bit of variety! :)

**Pull request merged**